python/GafferImageUI/ImageWriterUI.py

```diff
@@ -114,6 +114,72 @@
 			"plugValueWidget:type", "GafferUI.PresetsPlugValueWidget",
 		],
 
+		"namingConvention" : [
+
+			"description",
+			"""
+			TODO
+			""",
+
+			"plugValueWidget:type", "GafferUI.PresetsPlugValueWidget",
+			"presetsPlugValueWidget:allowCustom", True,
+
+		],
+		"namingConvention.viewName" : [
+
+			"description",
+			"""
+			TODO
+			""",
+
+			"preset:Single Part", "${naming:view}",
+			"preset:Interleave Layers", "${naming:view}",
+			"preset:Standard", "${naming:view}",
+			"preset:Legacy", "${naming:view}",
+			"preset:Nuke", "${naming:view}",
+			"preset:Nuke ( Interleave Layers )", "${naming:view}",
+			"preset:Nuke ( Interleave Views )", "${naming:view}",
+
+			"userDefault", "${naming:view}",
```

Comment:
I was thinking we would need to default to legacy mode, and need a user default to set it to Standard ( this only really affects the channelName plug, but it felt more consistent to set a user default for all 3 - I can omit the user default on view and part name if you want me to.  Or should we not worry about backwards compatibility, and just switch old writers to writing standards compliant exrs by default?

Reply:
Just recording the outcome of offline conversations : we've decided to break compatibility now rather than later, since we don't see any benefit in writing single-part files by default in this day and age. We've done the same for `ImageReader.channelInterpretation` so that it is compatible with files written by the ImageWriter by default. We are willing to reconsider if anyone hits problems in their pipeline during beta testing.